src/view/configuration/configuration.jsx

```diff
@@ -427,13 +433,27 @@ const Configuration = () => {
                           <h3>Privacy</h3>
 
                           <div className="u-gapTop">
-                            <InfoTipLayout tip="Queues privacy-sensitive work until the user opts in.">
-                              <WrappedField
-                                name={`instances.${index}.optInEnabled`}
-                                component={Checkbox}
-                                label="Enable Opt-In"
+                            <InfoTipLayout tip="The consent level when there is no consent cookie.">
```

Comment:
I think "cookie" should maybe be considered an implementation detail. What do you think about "The consent level to be used if the user has not previously provided consent preferences."